xgi/generators/uniform.py

```diff
@@ -346,6 +369,47 @@ def _index_to_edge(index, n, m):
     return [(index // (n**r) % n) for r in range(m - 1, -1, -1)]
 
 
+def _index_to_edge_comb(index, n, m):
+    """Generate a hyperedge given an index in the list of possible edges.
```

Comment:
given the index, how is the hyperedge generated?

Reply:
Fixed.